src/kedro_vizro_labeling/dashboards/components.py

```diff
@@ -125,7 +128,8 @@ def build(self):
 
         return dbc.DropdownMenuItem(**(defaults | self.extra))
 
-
+# AM comment: possibly if we implemented this in Vizro we would simplify a bit to as a single model
+# DropdownMenu that takes a list of text/href or similar rather than list[DropdownMenuItem]. Not sure though.
```

Comment:
I see. Indeed, that would be simpler. The reason I did it like this is to be able to pass things in the `extra` field of each `DropdownMenuItem`. I had especially `divider` in mind.

Reply:
Ah ok. The way you configure `divider/header` functionality in `dbc.DropdownMenu` seems quite strange to me, not sure why they do it that way really 🤔 Generally with Vizro we attempt to simplify/abstract things into more "user-friendly" configuration, e.g. maybe a user could write `items=["a", -1, "b"]` where `-1` would put a divider in there or similar.

As elsewhere, this is just FYI about the "Vizro way" of doing things. If you're just writing something for your own use then it really doesn't matter, and you can configure the models however it makes sense for you.

Reply:
Makes much more sense, I'll do that :)

Reply:
Actually, I just remembered, another reason why I was making a vizro model for DropdownMenuItem is the possibility to associate actions with each individual item. Would there be a vizro way to do this? :)